### PR TITLE
Switch CI to GHA.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,12 +20,12 @@ def read(*path):
     author_email='mail@gocept.com',
     description='Creates and drops temporary databases for testing purposes.',
     long_description=(
-        read('README.txt') +
-        read('COPYRIGHT.txt') +
+        read('README.rst') +
+        read('COPYRIGHT.rst') +
```
Comment (sallner) [inline setup.py:24]: Typically, the copyright is not necessarily formatted as RST, but it does not hurt here I guess.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+name: tests
```
Comment (sallner) [inline .github/workflows/tests.yml:1]: If we are at GHA now, we could get rid rid of .travis.yml.

**Review by sallner — APPROVED** [review]
Some minor comments.

File: .github/workflows/tests.yml (hunk 2 above)
Comment (icemac) [reply .github/workflows/tests.yml:1]: Good catch.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**